pages/create.js

```diff
@@ -35,20 +39,17 @@ function FormPage({ defaultTags }) {
     }
   }
 
+  useEffect(() => {
+    if (submitted) {
+      notify();
+    }
+  }, [submitted]);
+
```

Comment:
Again, like the edit page, i think we can move the call of the `notify` function if we can back a good response from the backend. 

i.e.

``` js
if (response.ok) {
  notify(); // <- replace the setSubmitted call with notify
  setError(null);
  mutate();
  event.target.reset();
  event.target.elements.name.focus();
} else { ...

```